[0035] Remove redundant bias interpretation type (#901)
microsoft/hlsl-specs: proposals/0035-linalg-matrix.md

Validation will enforce that:
* The input vector length matches the `K` matrix dimension
* The bias vector and output vector length matches the `M` matrix dimension

> * The bias vector's element type matches the output vector type.

Reviewer: and this one:)
```suggestion
* The bias vector's element type matches the output vector's element type.
```

* The matrix A is an `A` matrix of `Thread` scope

> * The input and bias interpretation type must be one of the valid linalg

Reviewer: Sorry for the late comment! I just noticed this

```suggestion
* The input must be one of the valid linalg
```

Validation will enforce that:
* The input vector length matches the `K` matrix dimension
* The bias vector length matches the `M` matrix dimension

> * The bias vector's element type and length must match the output vector type.

Reviewer 2: Is it weird to mention length again in this statement when we already say it must match `M` matrix dimension directly above? So, this could just be 'The bias vector's element type must match the output vector type' ?

Author: I made a small edit to the bullet above and this to make it more clear.

> * The bias vector's element type and length must match the output vector type.

Author: ```suggestion
* The bias vector and output vector length matches the `M` matrix dimension
* The bias vector's element type matches the output vector type.
```
